server/locale/DE/GPL/styles/flex/ratio--1x1.css

```
@@ -0,0 +1,48 @@
+@media (max-aspect-ratio: 11/10) {
+    .message__headline span.multi:nth-of-type(1) {
+        display: none;
+    }
+    .message__headline span.multi:nth-of-type(2) {
+        display: inline;
+    }
+    .message__headline {
+        font-size: 8vw;
+        line-height: 1.33em;
+    }
+    .message__disclaimer {
+        font-size: 0.9rem;
+        line-height: 6.66vw;
+        margin: 6% 0;
+    }
+    .message__headline span:nth-of-type(1) {
+        margin-right: -4px;
+    }
+    .message__logo-container {
+        width: 100%;
+    }
+    .message__logo:nth-of-type(1) {
+        width: 27px;
+        max-width: 15%;
+    }
+    .message__logo:nth-of-type(2) {
+        width: 91px;
+        max-width: 45%;
+    }
+}
+@media (max-aspect-ratio: 11/10) and (max-width: 220px) {
+    .message__headline {
+        font-size: 10vw;
+    }
+    .message__headline span.multi:nth-of-type(1) {
+        display: inline;
+    }
+    .message__headline span.multi:nth-of-type(2) {
+        display: none;
+    }
+}
+
+@media (max-aspect-ratio: 11/10) and (max-width: 150px) {
+    .message__headline {
+        font-size: 0.9rem;
+    }
```

After the closing curly brace for the ```.message__headline``` rule, I think we can add this:
```suggestion
    }
    .message__disclaimer {
        font-size: 9.5px;
    }
```